rabbitmq/README.md

```diff
@@ -76,6 +76,7 @@ Configuration Options:
 | Option                           | Required | Description                                                                                                                                                                                                                                                                                                                                                                                                 |
 |----------------------------------|----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `rabbitmq_api_url`               | Yes      | Points to the API url of the [RabbitMQ Managment Plugin][9].                                                                                                                                                                                                                                                                                                                                                |
+| `tls_verify`               | No      |  Set to false to skip verification of tls cert chain when the `rabbitmq_api_url` uses https. The default is true                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
switched from ssl_verify to tls_verify.

Reply:
turns out the tls_verify addition to the example config file was made recently. so all that's needed is the addition of this (and possibly others that are not documented here) to the README doc